hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelineManager.java

```diff
@@ -75,4 +75,6 @@ void finalizeAndDestroyPipeline(Pipeline pipeline, boolean onTimeout)
   void startPipelineCreator();
 
   void triggerPipelineCreation();
+
+  PipelineFactory getPipelineFactory();
```

Comment:
We don't need to expose this api as well. The shutdown for pipelineFactory should be called from PipelineManager itself.

Reply:
Done